scripts/trilinos-integration/white_run_jenkins_script_cuda-pascal

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -el
+ulimit -c 0
+
+KOKKOSKERNELS_BRANCH=$1
+TRILINOS_UPDATE_BRANCH=$2
+TRILINOS_PRISTINE_BRANCH=$3
+KOKKOS_UPDATE_BRANCH=$4
+
+if [ -z $KOKKOSKERNELS_BRANCH ]
+then
+  KOKKOSKERNELS_BRANCH=develop
+fi
+
+if [ -z $TRILINOS_UPDATE_BRANCH ]
+then
+  TRILINOS_UPDATE_BRANCH=develop
+fi
+
+if [ -z $TRILINOS_PRISTINE_BRANCH ]
+then
+  TRILINOS_PRISTINE_BRANCH=develop
+fi
+
+module load devpack/20180521/openmpi/2.1.2/gcc/7.2.0/cuda/9.2.88
+module swap openblas/0.2.20/gcc/7.2.0 netlib/3.8.0/gcc/7.2.0
+# Trilinos now requires cmake version >= 3.10.0
+module swap cmake/3.9.6 cmake/3.12.3
+export OMP_NUM_THREADS=8
+export JENKINS_DO_CUDA=ON
+export JENKINS_DO_OPENMP=OFF
+export JENKINS_DO_PTHREAD=OFF
+export JENKINS_DO_SERIAL=ON
+export JENKINS_DO_COMPLEX=OFF
+
+export JENKINS_ARCH="Power8,Pascal60"
+export JENKINS_ARCH_CXX_FLAG="-mcpu=power8 -arch=sm_60 --expt-extended-lambda --std=c++11"
+export JENKINS_ARCH_C_FLAG="-mcpu=power8"
```

Comment:
is this manual setup for cflags necessary ? it should be set from cmake script ?

Reply:
I'm not sure how/if the KOKKOS_ARCH info propogates to the cflags, this may be a remnant that can be removed but was in the scripts when Christian updated them after introducing KOKKOS_ARCH so I left it in.

Reply:
I guessed so. However, as this is trilinos integration test script, those environment variable should be caught from trilinos cmake script. If we provide correct environment, then we cannot test trilinos configure is right or not. While integrating trilinos, this problem is not important but it would be good to be addressed in my opinion.